src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -103,6 +103,83 @@ def validate_client_parameters(namespace):
         n.account_key = _query_account_key(n.account_name)
 
 
+def process_blob_source_uri(namespace):
```

Comment:
We have an even more complex logic which will deal with a source in both file share and blob container. This is a slimmed down version for blob container source only.